ospfd/ospf_sr.c

```diff
@@ -2112,154 +2107,180 @@ static int ospf_sr_enabled(struct vty *vty)
 	return 0;
 }
 
-/**
- * Update SRGB following new CLI value.
- *
- * @param lower	Lower bound of the SRGB
- * @param size	Size of the SRGB
- *
- * @return	0 on success, -1 otherwise
- */
```

Comment:
Please don't remove doxygen doc, instead update it accordingly.

Reply:
my bad, I did not realize I had deleted this - I will add doxygen docs for the updated function